### PR TITLE
Adding a set of test wsdls and corresponding golden output OAS files.

### PR DESCRIPTION
**Pull request by gideongoodwin**
These test cases represent some bugs found in customer docs. The corresponding unit tests are all failing (disabled). Next step is to fix the conversion logic being exercised here.

### CODE REVIEW

File: src/test/resources/spec-conversion/b-112274566_oas.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        "description": "Implements WSDL operation testOperation",
+        "parameters": []
+      }
+    }
+  },
+  "definitions": {
+    "testResponse": {
+        "properties": {
+            "field3": {
+                "type": "string"
```
[the remark this answers is not in the document]
Comment (gideongoodwin) [reply src/test/resources/spec-conversion/b-112274566_oas.json]: Fixed

File: src/test/resources/spec-conversion/b-112274566_oas.json [hunk 2]
```diff
@@ -0,0 +1,66 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "testService",
+    "description": "A OAS document generated from WSDL",
```
Comment (clusterwin) [inline src/test/resources/spec-conversion/b-112274566_oas.json]: An OAS

File: src/test/resources/spec-conversion/b-112274526_oas.json [hunk 3]
```diff
@@ -0,0 +1,67 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "testService",
+    "description": "A OAS document generated from WSDL",
```
Comment (clusterwin) [inline src/test/resources/spec-conversion/b-112274526_oas.json]: An OAS

File: src/test/resources/spec-conversion/b-112274847_oas.json [hunk 4]
```diff
@@ -0,0 +1,66 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "testService",
+    "description": "A OAS document generated from WSDL",
```
Comment (clusterwin) [inline src/test/resources/spec-conversion/b-112274847_oas.json]: An OAS

File: src/test/resources/spec-conversion/b-112275181_oas.json [hunk 5]
```diff
@@ -0,0 +1,69 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "testService",
+    "description": "A OAS document generated from WSDL",
```
Comment (clusterwin) [inline src/test/resources/spec-conversion/b-112275181_oas.json]: An OAS

File: src/test/resources/spec-conversion/b-112274566_oas.json (hunk 2 above)
Comment (gideongoodwin) [reply src/test/resources/spec-conversion/b-112274566_oas.json]: Fixed

File: src/test/resources/spec-conversion/b-112274526_oas.json (hunk 3 above)
Comment (gideongoodwin) [reply src/test/resources/spec-conversion/b-112274526_oas.json]: Fixed

File: src/test/resources/spec-conversion/b-112274847_oas.json (hunk 4 above)
Comment (gideongoodwin) [reply src/test/resources/spec-conversion/b-112274847_oas.json]: Fixed

File: src/test/resources/spec-conversion/b-112275181_oas.json (hunk 5 above)
Comment (gideongoodwin) [reply src/test/resources/spec-conversion/b-112275181_oas.json]: Fixed

**Pull request merged**